### ISSUE TITLE
Make Test fails under Xcode 13.3 (macOS 12.3).

### ISSUE THREAD

**Issue by eviltofu** [AlphaWallet/alpha-wallet-ios#4091]
`make test` produces the following error:
```
xcodebuild: error: Unable to find a destination matching the provided destination specifier:
		{ platform:iOS Simulator, OS:15.2, name:iPhone 12 }

	The requested device could not be found because no available devices matched the request.

	Available destinations for the "AlphaWallet" scheme:
		{ platform:macOS, arch:arm64, variant:Designed for [iPad,iPhone], id:00008103-000D45A126A0801E }
		{ platform:iOS, id:00008030-00184DAA3CDB402E, name:Hyperion }
		{ platform:iOS Simulator, id:AC32E650-70CB-42A1-BF0F-995947D16B07, OS:13.7, name:iPad (7th generation) }
		{ platform:iOS Simulator, id:8C64C21C-F73F-47DC-99F1-30E95274654D, OS:14.5, name:iPad (8th generation) }
		{ platform:iOS Simulator, id:539AD432-3A0C-47D7-B17E-4B73864438FB, OS:15.4, name:iPad (9th generation) }
		{ platform:iOS Simulator, id:ACD4F57A-3ED4-4857-BB5E-1E12A6C9AC76, OS:13.7, name:iPad Air (3rd generation) }
		{ platform:iOS Simulator, id:472F5FC1-127F-45AE-9D8A-48478EDA662C, OS:14.5, name:iPad Air (4th generation) }
		{ platform:iOS Simulator, id:FDACBEE8-5DE6-4A52-B61F-C8A3800F3458, OS:15.4, name:iPad Air (5th generation) }
		{ platform:iOS Simulator, id:347B6543-1646-4CAC-A9E0-29A8DFF1A856, OS:13.7, name:iPad Pro (9.7-inch) }
		{ platform:iOS Simulator, id:32DBDF67-B14F-4A70-9E6D-9D885AA668BA, OS:14.5, name:iPad Pro (9.7-inch) }
		{ platform:iOS Simulator, id:E1355E64-CD06-4237-8A75-8CDB0403908C, OS:15.4, name:iPad Pro (9.7-inch) }
		{ platform:iOS Simulator, id:F00E28C6-23A3-414D-8F3F-33BD9A6FACAD, OS:13.7, name:iPad Pro (11-inch) (2nd generation) }
		{ platform:iOS Simulator, id:4A842CA1-1AFC-4820-9302-65D61591B442, OS:14.5, name:iPad Pro (11-inch) (3rd generation) }
		{ platform:iOS Simulator, id:753E2A80-2942-414B-8526-0E3D3A600F72, OS:15.4, name:iPad Pro (11-inch) (3rd generation) }
		{ platform:iOS Simulator, id:C6B5E891-BA8D-488D-94DE-DAECC09061E5, OS:13.7, name:iPad Pro (12.9-inch) (4th generation) }
		{ platform:iOS Simulator, id:F6D788F8-5898-4BAE-AFA7-1226364CE2A2, OS:14.5, name:iPad Pro (12.9-inch) (5th generation) }
		{ platform:iOS Simulator, id:FFDD6817-11A1-418C-87CA-1BC4410E0672, OS:15.4, name:iPad Pro (12.9-inch) (5th generation) }
		{ platform:iOS Simulator, id:6BAC142C-CB9A-48B7-BC92-83A533A91E04, OS:15.4, name:iPad mini (6th generation) }
		{ platform:iOS Simulator, id:9D9D62D8-6E2F-4D02-8170-416776053368, OS:13.7, name:iPhone 8 }
		{ platform:iOS Simulator, id:A218E3EA-0A49-4FD3-A94E-0C3BE2B6A7E4, OS:14.5, name:iPhone 8 }
		{ platform:iOS Simulator, id:78CCAFEA-7D36-449D-AFEA-62FF33F89274, OS:15.4, name:iPhone 8 }
		{ platform:iOS Simulator, id:C702EB8F-8770-4F12-8DC8-B88831A8E221, OS:13.7, name:iPhone 8 Plus }
		{ platform:iOS Simulator, id:C77A2CC7-DB88-4A89-91F7-C36D9F7C8F27, OS:14.5, name:iPhone 8 Plus }
		{ platform:iOS Simulator, id:3E257E29-AE4D-45EE-A094-2A4FFB76652B, OS:15.4, name:iPhone 8 Plus }
		{ platform:iOS Simulator, id:7C21B721-1363-4C61-A325-3FB0DC0576D6, OS:13.7, name:iPhone 11 }
		{ platform:iOS Simulator, id:752E5FDF-F5E2-4E94-A15A-3C5F3BD43B8B, OS:14.5, name:iPhone 11 }
		{ platform:iOS Simulator, id:1CFDF664-E4CC-4664-A410-0CA2D307ED7E, OS:15.4, name:iPhone 11 }
		{ platform:iOS Simulator, id:37C886CC-4363-4887-A791-AD08CBB7610F, OS:13.7, name:iPhone 11 Pro }
		{ platform:iOS Simulator, id:3AFE56CA-A745-4619-9AA2-C583E20013F1, OS:14.5, name:iPhone 11 Pro }
		{ platform:iOS Simulator, id:FD651269-185D-448B-AECB-5E82453713B7, OS:15.4, name:iPhone 11 Pro }
		{ platform:iOS Simulator, id:59749576-61AD-4803-9629-4275819504AD, OS:13.7, name:iPhone 11 Pro Max }
		{ platform:iOS Simulator, id:7A4C086E-54B4-4E63-BADD-1799A3A72F75, OS:14.5, name:iPhone 11 Pro Max }
		{ platform:iOS Simulator, id:33292075-4009-492F-9C2B-508D60266520, OS:15.4, name:iPhone 11 Pro Max }
		{ platform:iOS Simulator, id:179FF62D-3B0D-41CD-BC2A-E084DA22E2CC, OS:14.5, name:iPhone 12 }
		{ platform:iOS Simulator, id:FC5E81E1-4234-4A43-8765-C4F30E3BAB3A, OS:15.4, name:iPhone 12 }
		{ platform:iOS Simulator, id:38B4486A-0A76-46C4-86E7-8DA904A0283E, OS:14.5, name:iPhone 12 Pro }
		{ platform:iOS Simulator, id:FFCCB61F-128A-4C7B-B023-B8211DEF2FF8, OS:15.4, name:iPhone 12 Pro }
		{ platform:iOS Simulator, id:A172D7B4-8C97-4F03-B863-F1967D8AE510, OS:14.5, name:iPhone 12 Pro Max }
		{ platform:iOS Simulator, id:ADC5A2ED-C2F1-4BEC-AF6A-C26C93D011C7, OS:15.4, name:iPhone 12 Pro Max }
		{ platform:iOS Simulator, id:84A1D8FF-BA97-4487-86B4-B2C15BD65DFF, OS:14.5, name:iPhone 12 mini }
		{ platform:iOS Simulator, id:BBE728E1-EF5D-44FC-95B9-8B7C726880C8, OS:15.4, name:iPhone 12 mini }
		{ platform:iOS Simulator, id:CC1BEF36-D9D1-4635-8313-432D08BBED9A, OS:15.4, name:iPhone 13 }
		{ platform:iOS Simulator, id:9E719100-19AF-4B0A-8491-4AF7170C1541, OS:15.4, name:iPhone 13 Pro }
		{ platform:iOS Simulator, id:4DB65D20-4C38-4E53-BA71-1B2164CCD88E, OS:15.4, name:iPhone 13 Pro Max }
		{ platform:iOS Simulator, id:DD3B7214-5434-405B-A7F4-FD2357E21D9C, OS:15.4, name:iPhone 13 mini }
		{ platform:iOS Simulator, id:968CB5A5-6233-4CEB-8754-2BFC6DC13D54, OS:13.7, name:iPhone SE (2nd generation) }
		{ platform:iOS Simulator, id:CBD326F8-567C-416E-9528-07011E0E0D53, OS:14.5, name:iPhone SE (2nd generation) }
		{ platform:iOS Simulator, id:ADFBD85E-4C9D-4464-B9FB-8BF235ADFAA9, OS:15.4, name:iPhone SE (3rd generation) }
		{ platform:iOS Simulator, id:77064335-48B9-4498-BC6B-71980F9964ED, OS:14.5, name:iPod touch (7th generation) }
		{ platform:iOS Simulator, id:E2D11593-AF4F-45DD-893E-7B07CAE46971, OS:15.4, name:iPod touch (7th generation) }

	Ineligible destinations for the "AlphaWallet" scheme:
		{ platform:iOS, id:dvtdevice-DVTiPhonePlaceholder-iphoneos:placeholder, name:Any iOS Device }
		{ platform:iOS Simulator, id:dvtdevice-DVTiOSDeviceSimulatorPlaceholder-iphonesimulator:placeholder, name:Any iOS Simulator Device }
```

**Comment by eviltofu** [comment]
One option is to download the 15.2 simulators but I think we should have the tests work right out of the box.

**Comment by eviltofu** [comment]
Closed due to [DEV] Update Xcode version to use since Xcode 13.3.1 uses an incompatible Package.resolved file format #4309
